services/graphite/service.go

```
@@ -46,16 +46,21 @@ type Service struct {
 // NewService returns an instance of the Graphite service.
 func NewService(c Config) *Service {
 	s := Service{
-		bindAddress:      c.BindAddress,
-		database:         c.Database,
-		protocol:         c.Protocol,
-		batchSize:        c.BatchSize,
-		batchTimeout:     time.Duration(c.BatchTimeout),
-		consistencyLevel: c.ConsistencyAsEnum(),
-		logger:           log.New(os.Stderr, "[graphite] ", log.LstdFlags),
-		done:             make(chan struct{}),
+		bindAddress:  c.BindAddress,
+		database:     c.Database,
+		protocol:     c.Protocol,
+		batchSize:    c.BatchSize,
+		batchTimeout: time.Duration(c.BatchTimeout),
+		logger:       log.New(os.Stderr, "[graphite] ", log.LstdFlags),
+		done:         make(chan struct{}),
 	}
 
+	consistencyLevel, err := cluster.ParseConsistencyLevel(c.ConsistencyLevel)
+	if err != nil {
+		return nil
+	}
+	s.consistencyLevel = consistencyLevel
```

I think you should return an error from `NewService()`:

```go
func NewService(c Config) (*Service, error)
```